website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -650,7 +678,7 @@ Properties without a default value are required.
 | `data.attributes.assessments-enabled`         | boolean        | `false`          | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `assessments-enforced`. Only available for Plus tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).  |
 | `data.attributes.auto-apply`                  | boolean        | (previous value) | Whether to automatically apply changes when a Terraform plan is successful, [with some exceptions](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                   |
 | `data.attributes.description`                 | string         | (previous value) | A description for the workspace.                                                                                                                                                                                                                                                                                                                                                                                   |
-| `data.attributes.execution-mode`              | string         | (previous value) | Which [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) to use. Valid values are `remote`, `local`, and `agent`. When set to `local`, the workspace will be used for state storage only. This value must not be specified if `operations` is specified.                                                                                                                                             |
+| `data.attributes.execution-mode`              | string         | (previous value) | Which [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) to use. Valid values are `default`, remote`, `local`, and `agent`. When set to `local`, the workspace will be used for state storage only. This value must not be specified if `operations` is specified.                                                                                                                                             |
```

Comment:
Similar comment here, passing default directly is not allowed

Reply:
Also, I think we will need to add the `setting-overwrites` attributes to the create/update sections as well.
Just spitballing, but it may be worthwhile adding a subsection about "Workspace Setting Overwrites".

Alternatively (or maybe in addition to that), we could add a new page for "Setting Overwrites" under https://developer.hashicorp.com/terraform/cloud-docs/api-docs (like the overview page), which explains how the setting overwrites work in the API in general

I think we definitely need to tell users "here are the attributes which you can overwrite for the workspace" (at this point just execution mode and agent pool)...how we want to describe the format of the new `setting-overwrites` in general, which is a whole new API concept, is something we could definitely use feedback from @rkoron007 on